src/objects/inline_keyboard.rs

```diff
@@ -0,0 +1,39 @@
+use std::option::Option;
+
+#[derive(Serialize, Debug)]
+pub struct Markup {
+	pub inline_keyboard: Vec<Vec<Button>>,
+}
+
+#[derive(Serialize, Debug)]
+pub struct Button {
```

Comment:
The name of this in the Telegram bot API is "InlineKeyboardButton", please try to keep the names as close as possible.